pyrigi/data_type.py

```
@@ -50,20 +51,29 @@ def point_to_vector(point: Point) -> Matrix:
     """
     Return point as single column sympy Matrix.
     """
-
-    if isinstance(point, MatrixBase):
-        if point.shape[1] != 1:
+    if isinstance(point, MatrixBase) or isinstance(point, np.ndarray):
+        if (
+            len(point.shape) > 1 and point.shape[0] != 1 and point.shape[1] != 1
+        ) or len(point.shape) > 2:
             raise ValueError("Point could not be interpreted as column vector.")
-        return point
+        if isinstance(point, np.ndarray):
+            point = np.array([point]) if len(point.shape) == 1 else point
+            point = Matrix(
+                [
+                    [float(point[i, j]) for i in range(point.shape[0])]
+                    for j in range(point.shape[1])
+                ]
+            )
+        return point if (point.shape[1] == 1) else point.transpose()
 
     if not isinstance(point, Sequence) or isinstance(point, str):
         raise TypeError("The point must be a Sequence of Coordinates.")
 
     try:
         res = Matrix(point)
-    except SympifyError as e:
+    except Exception as e:
```

We should catch all possible exceptions that come about when converting to `Matrix`. Especially, because we cannot possibly catch all exceptions in the lines before.